code/datums/datumvars.dm

```diff
@@ -414,7 +414,7 @@ body
 				html += "</ul>"

```

Comment:
You can't see it without expanding the diff, but a few lines above this shows how viewing a list's contents recursively calls this same proc.